core/reporter-api/src/main/java/org/arquillian/reporter/api/model/entry/FileEntry.java

```diff
@@ -1,9 +1,40 @@
 package org.arquillian.reporter.api.model.entry;
 
+import java.nio.file.Path;
+
 /**
  * An {@link Entry} representing a file
  *
  * @author <a href="mailto:mjobanek@redhat.com">Matous Jobanek</a>
  */
 public class FileEntry implements Entry {
```

Comment:
Not sure if it should be called PathEntry or we should change Path to File, but for me Path is better to use for the operations it offers.